src/GravitationalClustering.js

```diff
@@ -65,6 +48,7 @@ export default class GravitationalClustering {
         this.eps = Number.MAX_VALUE;
         this.engine = Random.engines.mt19937();
         this.engine.seed(this.seed);
```

Comment:
I prefer to let the user provide the random function (with a default of Math.random) instead of the seed